etc/adminhtml/system.xml

```diff
@@ -394,6 +394,15 @@
                         <config_path>payment/boltpay/new_plugin_version_notification</config_path>
                         <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                     </field>
+                    <field id="instant_button_variant" translate="label" type="text" sortOrder="390" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Bolt button variant</label>
+                        <config_path>payment/boltpay/instant_button_variant</config_path>
+                        <comment>Add </comment>
```

Comment:
Nit: fix comment
Is there additional information we may provide to the merchants? Maybe link to our documentation?

Reply:
ugh I was typing and forgot to add the rest adding now. 

Currently theres no way for merchants to create variants, we create it for them so im not sure we can have a public doc for this. Ill follow up with support in case they have something.

Reply:
Once we build the functionality to edit variants in the merchant dashboard we can add some links here untill then its not much use for now just adding a comment on what its for.